### ISSUE TITLE
Add a `kitchen sink` of options to the Filtered text format

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#1520]
As we discussed over in https://github.com/backdrop/backdrop-issues/issues/1091, we think new users might be expecting a lot more buttons in their rich-text editor, but having a gazillion buttons might be overwhelming, so we would like to hid them in a WordPress style "Kitchen Sink" button, that will hide / show the tools available.

**Comment by olafgrabienski** [comment]
Any progress here? I'm asking because I miss headings in the default HTML tags and CKeditor buttons for Filtered HTML. If there was the "kitchen sink", I'd be fine with it.

Slightly off topic: If I have to configure the text format defaults for every site, I'd propose to change them adding headings beginning with `h2` and the Format dropdown in CKeditor. (Didn't find an issue where the defaults are discussed. Did I miss it? If not, I'm happy to open one.)

**Comment by jenlampton** [comment]
>  I'd propose to change them adding headings beginning with h2 and the Format dropdown in CKeditor. (Didn't find an issue where the defaults are discussed. Did I miss it? If not, I'm happy to open one.)

Hm, I swear we used to have the "Formats" option available by default. This could be another bug.  Could you please open a new issue, add the label "Bug" and add the milestone 1.4.5 :)

**Comment by olafgrabienski** [comment]
To move forward: Which buttons should we add to the 'kitchen sink'?

For reference, here a screenshot of a current WordPress installation:

![screen-wordpress-editor-buttons](https://user-images.githubusercontent.com/14188307/27254486-821b0600-5389-11e7-80a5-a95b60039937.png)

The second row is the 'kitchen sink', including (from left to right):

- strikethrough text
- add a horizontal line
- choose text color
- paste as text
- delete formatting
- add special characters
- reduce indention
- increase indention
- undo
- repeat
- display a list of shortcuts

**Comment by olafgrabienski** [comment]
As a reminder: depending on the buttons we add, we might have to add some 'allowed HTML tags' in the text format configuration as well.

**Comment by jenlampton** [comment]
We should avoid using inline styles, but if we decide that we want to allow text color changes (IMHO it's not worth pursuing) it might be smarter to try something with classes or data attributes as to not muck up the content by adding in styles inline.

**Comment by olafgrabienski** [comment]
Yes, we should avoid inline styles. That said, where would we define color classes to make them theme independent: in a CSS file belonging to core / CKEditor?
